src/cli/shell.rs

```diff
@@ -159,9 +159,10 @@ async fn start_unix_shell<T: Shell + Copy + 'static>(
 
     // Write custom prompt to the env file
     temp_file.write(prompt.as_bytes()).into_diagnostic()?;
+    temp_file.flush().into_diagnostic()?;
```

Comment:
Should this be done for the other shells?

Reply:
Done!